packages/dd-trace/src/ritm.js

```diff
@@ -50,8 +50,20 @@ function Hook (modules, options, onrequire) {
 
   if (patchedRequire) return
 
+  const _origRequire = Module.prototype.require
```

Comment:
this might be unrelated to this PR, but if we are using this `_origRequire`, then can we remove the `origRequire` above and use this one in other places too?